### DISCUSSION TITLE
Can Kyuubi be deployed in Docker?

### DISCUSSION THREAD

**Discussion by FN20200222** [apache/kyuubi#6654] [Q&A]
有docker部署吗，我们服务器基于docker的

**Comment by pan3793** [comment]
Convert to a discussion as this is a question instead of an issue. For more details, please read [FAQ](https://github.com/apache/kyuubi/discussions/2481)
> We encourage users to ask question in [Mailing List](https://kyuubi.apache.org/mailing_lists.html) and [GitHub Discussions](https://github.com/apache/incubator-kyuubi/discussions), and report bugs in [GitHub Issues](https://github.com/apache/incubator-kyuubi/issues)

> 有docker部署吗
> (Translation: Can Kyuubi be deployed in Docker?)

The short answer is yes. Kyuubi is a normal JVM-based software, nothing blocks you run it inside the container, and you can easily get the official Docker image from [Docker Hub](https://hub.docker.com/r/apache/kyuubi).

But as Kyuubi requires engines, e.g. Spark or Flink, to run your workloads, you need to read the docs to configure those components properly inside the container, this part is beyond how to run Kyuubi inside the Docker.
